mxcube3/routes/Queue.py

```diff
@@ -383,6 +384,12 @@ def add_centring(id):
     return resp
 
 
+@mxcube.route("/mxcube/api/v0.1/results/<int:number>")
+def serve_char_result(number):
```

Comment:
Its maybe better to have a function that just returns the result as HTML or JSON, instead of serving the file statically.  Because the results will later be stored in LIMS.